dodola/cli.py

```diff
@@ -51,6 +51,27 @@ def dodola_cli(debug):
     logging.basicConfig(level=loglevel)
 
 
+@dodola_cli.command(help="Clean up and standardize GCM")
+@click.argument("x", required=True)
+@click.argument("out", required=True)
+@click.option(
+    "--drop_leapdays/--no-drop_leapdays",
```

Comment:
Suggest changing
`"--drop_leapdays/--no-drop_leapdays"` to `"--drop-leapdays/--no-drop-leapdays"`- replacing underscores. This makes it a bit more consistent. Underscores also sometimes don't render properly in terminals, this is a problem if it's a command option or arg, so convention is to generally stick to hyphens in CLI options (esp in POSIX-like systems).

I think it will still pass to the `drop_leapdays` kwargs in `cleancmip6()`, even if you give it a hyphen for the CLI option.